src/libs/Navigation/AppNavigator/ModalStackNavigators.tsx

```diff
@@ -203,7 +203,7 @@ const SettingsModalStackNavigator = createModalStackNavigator<SettingsNavigatorP
     [SCREENS.SETTINGS.PREFERENCES.THEME]: () => require('../../../pages/settings/Preferences/ThemePage').default as React.ComponentType,
     [SCREENS.SETTINGS.CLOSE]: () => require('../../../pages/settings/Security/CloseAccountPage').default as React.ComponentType,
     [SCREENS.SETTINGS.SECURITY]: () => require('../../../pages/settings/Security/SecuritySettingsPage').default as React.ComponentType,
-    [SCREENS.SETTINGS.ABOUT]: () => require('../../../pages/settings/AboutPage/AboutPage').default as React.ComponentType,
+    [SCREENS.SETTINGS.ABOUT]: () => require('../../../pages/settings/AboutPage').default as React.ComponentType,
```

Comment:
Not sure how this isn't an error on main... 🤔

Reply:
![image](https://github.com/Expensify/App/assets/46095609/2562e269-a7d5-4aac-9703-39bd3a1efb22)

You changed the structure in this PR, right?